agu_oss/clean_data.py

```diff
@@ -1,6 +1,7 @@
 """A second sample module to clean data."""
 
 import pandas as pd
+import numpy as np
```

Comment:
Great job submitting this pr @Cecile-A-C !

Reply:
Hey, why did you add this impirort?